### ISSUE TITLE
Plotting function: incorrect graph

### ISSUE THREAD

**Issue by JuniorDelaure** [numworks/epsilon#1350]
Version 12.4.0 
This function's graph is incorrect 

x^2.0000001

![IMG_20200224_094912](https://user-images.githubusercontent.com/56046513/75138799-68932b00-56eb-11ea-92b6-d87aa9e7f757.jpg)

![IMG_20200224_094852](https://user-images.githubusercontent.com/56046513/75138788-6204b380-56eb-11ea-9e32-c06d10c45eff.jpg)

**Comment by nsajko** [comment]
This is kind of maybe similar to how second-derivatives (of any function, I think) get graphed. That is, the *steps* are too large in that case, too.

**Comment by EmilieNumworks** [comment]
Good catch @JuniorDelaure, thanks! There are indeed two issues here: first, the function should not be defined for x < 0 as its imaginary part is not null and, second, there is a major precision loss on the function plotting...

1. When computing power of complexes, we sometimes neglect the imaginary part or the real part to avoid weird result as `exp(iπ) = -1+6E-17*i`. However, the decision to neglect is based on an absolute comparison when it should depends on the values of the complexes (Cf comment and code in `poincare/src/power.cpp` on method  `PowerNode::compute`). For instance, (-10)^0.00000001 ~ 1 when it should be ~1+3.141593ᴇ-8*i

2. When simplifying x^(p/q), with p, q integers, before an approximation, we turn it into (nth_root(x,q))^p. This was done in order to be able to choose another root than the principal root if the complex mode is real. For instance, (-8)^(1/3) returns -2 in real mode and 1+sqrt(3)*i in cartesian mode. However, in our case, f(x) = x^(2.0000001) = x^(20000001/10000000) = nth_root(x, 10000000)^20000001. This has the side effect of raising the precision error in nth_root(x, 10000000) to the power 20000001... This explains the weird look of the curve.

I've made 2 patches to fix these errors, I'll make a PR soon and refer it here.

**Comment by EmilieNumworks** [comment]
PR #1360 should resolve this issue.

**Comment by artaxxx** [comment]
![screenshot (52)](https://user-images.githubusercontent.com/20040032/78790856-680ad700-79af-11ea-8b0f-35224a667f30.png)
It's better now, but it's not perfect! The function isn't defined for x<0 but you can see a part of the graph...
